### ISSUE TITLE
orderedItems is not defined in properties section of vocabulary document

### ISSUE THREAD

**Issue by cwebber** [w3c/activitystreams#494]
https://www.w3.org/TR/activitystreams-vocabulary/#properties

**Comment by cjslep** [comment]
Dupe of #437 ?

**Comment by kefo** [comment]
Yes, this is for all intents and purposes a duplicate of 437.  437 is slightly broader, but gets at the same issue.

**Comment by evanp** [comment]
I think the correct behaviour here is to add a property to the vocabulary for `orderedItems`. I'm not sure this is an erratum, though. Maybe it's worth adding an ERRATA entry?
